yaml/boundaries.yaml

```diff
@@ -101,6 +101,13 @@ global:
           - geom_type: line
           - mz_boundary_from_polygon: true
 
+  - &dispute_properties
+    claimed_by: { col: claimed_by }
+    recognized_by: { col: recognized_by }
+    disputed_by: { col: disputed_by }
+    tz_admin_level: {col: admin_level }
+    dispute_id: {col: 'ne:brk_a3'}
```

Comment:
For our QA sanity, we should concat {col: 'ne:brk_a3'} and {col: 'ne_id'} together with a `_` separator for the disputed_id.

Reply:
`dispute_id` might deserve it's own global, since it's used separately a few places down below.

Reply:
Fixed in https://github.com/tilezen/vector-datasource/pull/2054/commits/6f75fc78a61e32afd27c6a36481a8d2ba9386bdf